### DISCUSSION TITLE
Enable SO_REUSEADDR (OSError: [Errno xx] Address already in use)

### DISCUSSION THREAD

**Discussion by azbub** [socketio/socket.io#4355] [Q&A] [answered]
I am subscribing to a wss feed and often times there is a disconnection from the server.

I want to automatically reconnect to the server so I added the following to my disconnect event : 

```
        @socketio_client.on('disconnect', namespace='/streaming')
        def on_disconnect():
            print('Server disconnected.')
            socketio_client.disconnect()
            os.system(f'python3 __scriptfile__py &')
            exit()
```

however when this event is triggered, I get this message : `OSError: [Errno 98] Address already in use`

I though I would either enable reuseaddr or kill the connection and start a new one, how does one proceed ?

**Comment by darrachequesne** [comment, marked as answer]
Hi! It seems you are using the Python client, could you please open an issue [there](https://github.com/miguelgrinberg/python-socketio)? 

Thanks!

**Reply by azbub** [reply]
Done.
